x-pack/plugins/spaces/index.js

```diff
@@ -47,12 +49,19 @@ export const spaces = (kibana) => new kibana.Plugin({
       };
     },
     replaceInjectedVars: async function (vars, request) {
+      // A rather obtuse way of preventing the Kibana login/logout resources from trying to make these requests.
```

Comment:
I'm open to suggestions here -- `replaceInjectedVars` is called when loading the login/logout pages, which throws visible security exceptions, because the user is not yet (or no longer) authenticated to ES.

We should only need to do the variable replacement when doing a full page load within Kibana, which I _believe_ is constrained to loading entire apps, via the `/app/*` pattern